README.md

```diff
@@ -114,6 +114,17 @@ There are multiple ways to run this sample: locally using Ollama or Azure OpenAI
 
 See the [cost estimation](./docs/cost.md) details for running this sample on Azure.
 
+#### (Optional) Enable additional user context to Microsoft Defender for Cloud
+In case you have Microsoft Defender for Cloud protection on your Azure OpenAI resource and you want to have additional context on the alerts, run this command:
+```bash
+azd env set MS_DEFENDER_ENABLED true
+```
+
+To customize the application name of the context, run this command:
+```bash
+azd env set APPLICATION_NAME <your application name>
```

Comment:
This name will be only visible in security Alerts or also in the Example Chat UX?

Reply:
Currently adding it only in the context